small tweaks to get docs ready for an open beta (#17542)
DataDog/documentation: content/en/database_monitoring/guide/connect_dbm_and_apm.md

{{% /tab %}}

{{% tab ".NET" %}}

<div class="alert alert-warning">

> This features requires automatic instrumentation to be enabled for your .NET service.

Reviewer: ```suggestion
This feature requires automatic instrumentation to be enabled for your .NET service.
```

</div>

Follow the [.NET Framework tracing instructions][1] or the [.NET Core tracing instructions][2] to install the automatic instrumentation package and enable tracing for your service.

> Ensure that you are using a supported client library (e.g. `Npgsql`).

Reviewer: ```suggestion
Ensure that you are using a supported client library. For example, `Npgsql`.
```

Enable the database monitoring propagation feature by setting the following environment variable:
   - `DD_DBM_PROPAGATION_MODE=full`

[1]: /tracing/trace_collection/dd_libraries/dotnet-framework
[2]: /tracing/trace_collection/dd_libraries/dotnet-core

{{% /tab %}}

{{% tab "PHP" %}}

<div class="alert alert-warning">

> This features requires the tracer extension to be enabled for your PHP service.

Reviewer: ```suggestion
This feature requires the tracer extension to be enabled for your PHP service.
```

</div>

Follow the [PHP tracing instructions][1] to install the automatic instrumentation package and enable tracing for your service.

Ensure that you are using a supported client library. For example, `PDO`.

Enable the database monitoring propagation feature by setting the following environment variable:
   - `DD_DBM_PROPAGATION_MODE=full`

[1]: https://docs.datadoghq.com/tracing/trace_collection/dd_libraries/php?tab=containers

{{% /tab %}}

{{% tab "Node.js" %}}

<div class="alert alert-warning">
DBM-APM linking for Node.js is in alpha release and may be unstable.
</div>

> Install or udpate [dd-trace-js][1] to version greater than `3.13.0` (or `2.22.0` if using end-of-life Node.js version 12):

Reviewer: ```suggestion
Install or update [dd-trace-js][1] to a version greater than `3.13.0` (or `2.22.0` if using end-of-life Node.js version 12):
```
